LICENCES-3RD-PARTY.txt

```diff
@@ -0,0 +1,15 @@
+Ckan_client-PHP
+---------------
+
+Copyright (c) 2010 Jeffrey Barke http://jeffreybarke.net/ https://github.com/jeffreybarke/Ckan_client-PHP Licensed under the MIT license Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated documentation files (the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions:
```

Comment:
A line break should be added before "Permission" (where the text of the license starts) for readability